.github/ISSUE_TEMPLATE.md

```diff
@@ -8,3 +18,4 @@ Thanks so much for filing an issue or feature request! Please fill out the follo
 * Repro steps and/or stack trace:

```

Comment:
Can probably remove this line, seeing as you added the `Reproduction` section. If you'd like, you could encourage them to post the stack trace in the `Description` section